analyzers/tests/SonarAnalyzer.UnitTest/Rules/PropertiesShouldBePreferredTest.cs

```diff
@@ -28,24 +28,27 @@ namespace SonarAnalyzer.UnitTest.Rules
     [TestClass]
     public class PropertiesShouldBePreferredTest
     {
+        private readonly VerifierBuilder builder = new VerifierBuilder<PropertiesShouldBePreferred>().AddReferences(MetadataReferenceFacade.SystemThreadingTasks);
```

Comment:
The `.AddReferences(MetadataReferenceFacade.SystemThreadingTasks)` part is needed only by one file, so it should be in the `PropertiesShouldBePreferred` test.

If there would be more tests using it (like a test + code fix test), than it would make sense here.